File: app/src/main/java/com/zhxh/rxjava2/samples/model/UserDetail.java
package com.zhxh.rxjava2.samples.model;

/**
 * Created by zhxh on 2018/1/18
 */

public class UserDetail {

    public UserDetail(long id) {
        this.id = id;
    }

    public long id;
    public String firstname;
    public String lastname;

    @Override
    public String toString() {
        return "UserDetail{" +
                "id=" + id +
                ", firstname='" + firstname + '\'' +
                ", lastname='" + lastname + '\'' +
                '}';
    }
}

File: app/src/main/java/com/zhxh/rxjava2/samples/model/UserDetail.kt
package com.zhxh.rxjava2.samples.model

/**
 * Created by zhxh on 2018/1/18
 */

class UserDetail(var id: Long) {
    var firstname: String? = null
    var lastname: String? = null

    override fun toString(): String {
        return "UserDetail{" +
            "id=" + id +
            ", firstname='" + firstname + '\''.toString() +
            ", lastname='" + lastname + '\''.toString() +
            '}'.toString()
    }
}
